engines/scumm/script_v5.cpp

```diff
@@ -2810,4 +2812,17 @@ void ScummEngine_v5::decodeParseString() {
 	_string[textSlot].saveDefault();
 }
 
+void ScummEngine_v5::printPatchedMI1CannibalString(int textSlot, const byte *ptr) {
+	const char *msg = (const char *)ptr;
+
+	if (strncmp((const char *)ptr, "/LH.ENG/", 8) == 0) {
+		msg =
+"Oooh, that's nice.\xFF\x03"
+"Simple.  Just like one of mine.\xFF\x03"
+"And little.  Like mine.";
```

Comment:
Is the double space after the period in this line and the line above intentional?

Reply:
> Is the double space after the period in this line and the line above intentional?

Yes, it's what I got from descumm with the floppy version's room-25-205s script:

```
[0086] (14) print(5,[Text("Take a look at this.")]);
[009E] (AE) WaitForMessage();
[00A0] (14) print(3,[Text("Oooh, that's nice." + wait() + "Simple.  Just like one of mine." + wait() + "And little.  Like mine.")]);
[00F0] (AE) WaitForMessage();
[00F2] (14) print(3,[Text("And it says, `Made by Lemonhead`^" + wait() + "^just like one of mine!" + wait() + "We should take this to the Great Monkey.")]);
[015A] (AE) WaitForMessage();
[015C] (14) print(5,[Text("Yes, I agree.")]);
[016D] (AE) WaitForMessage();
```

In the CD version, this was:

```
[0084] (14) print(5,[Text("Take a look at this.")]);
[009C] (AE) WaitForMessage();
[009E] (14) print(3,[Text("Oooh, that's nice.")]);
[00B4] (14) print(3,[Text("And it says, `Made by Lemonhead`^" + wait() + "^just like one of mine!" + wait() + "We should take this to the Great Monkey.")]);
[011C] (AE) WaitForMessage();
[011E] (14) print(5,[Text("Yes, I agree.")]);
[012F] (AE) WaitForMessage();
```